crates/matrix-sdk-crypto/src/store/memorystore.rs

```diff
@@ -292,7 +301,19 @@ impl CryptoStore for MemoryStore {
     }
 
     async fn load_backup_keys(&self) -> Result<BackupKeys> {
-        Ok(BackupKeys::default())
+        let stored = self.backup_keys.get("key");
+        Ok(match stored {
+            None => BackupKeys::default(),
+            Some(k) => {
+                let export = k.value();
+                // can't clone or copy it, so..
+                let key = export.recovery_key.as_ref().map(|e| serde_json::to_string(e).unwrap());
+                BackupKeys {
+                    recovery_key: key.map(|s| serde_json::from_str(&s).unwrap()),
```

Comment:
Heh, I had to add this in #2227 as well. Though I added just a clone to the backup key.

So let's do that instead.